docs/reference/README.md

```diff
@@ -0,0 +1,42 @@
+---
+title: 🔗 Reference Quick Links
+tags: [AvalancheGo APIs]
+description: AvalancheGo is the node implementation for the Avalanche network - a blockchain platform with high throughput and blazing fast transactions. This page is overview of the APIs associated with AvalancheGo. 
+sidebar_label: 🔗 Quick Links
+pagination_label: API Reference Quick Links
+---
+
+| AvalancheGo APIs       | Description  |
+| :------------------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**P-Chain API**](/reference/avalanchego/p-chain/api)      | Platform Chain API Reference |
+| [**C-Chain API**](/reference/avalanchego/c-chain/api)      | Contract Chain API Reference |
+| [**X-Chain API**](/reference/avalanchego/x-chain/api)      | Exchange Chain API Reference |
+| [**Admin API**](/reference/avalanchego/admin-api)      | Admin API Reference |
+| [**Auth API**](/reference/avalanchego/auth-api)      | Auth API Reference |
+| [**Health API**](/reference/avalanchego/health-api)      | Health API Reference |
+| [**Index API**](/reference/avalanchego/index-api)      | Index API Reference |
+| [**Info API**](/reference/avalanchego/info-api)      | Info API Reference |
+| [**IPC API**](/reference/avalanchego/ipc-api)      | IPC API Reference |
+| [**Keystore API**](/reference/avalanchego/keystore-api)      | Keystore API Reference |
+| [**Metrics API**](/reference/avalanchego/metrics-api)      | Metrics API Reference |
+
```

Comment:
Missing mentioning Subnet-EVM and its API

Reply:
good catch, added